### PR TITLE
Torso and Gimbal bringup

### PR DESCRIPTION
**Pull request by younesr1**

### CODE REVIEW

**Comment by younesr1** [comment]
This PR adds changes required to get the torso and gimbal moving. vids of torso and gimbal moving are posted to the discord. moveit_python is removed from movo_third_party and added to the rosinstall instead

File: movo_common/movo_ros/src/movo_jtas/movo_jtas.py [hunk 1]
```diff
@@ -34,7 +34,7 @@
  \Platform: Linux/ROS Indigo
 --------------------------------------------------------------------"""
 from movo_joint_interface.jaco_joint_controller import SIArmController
-from trajectory_smoother import TrajectorySmoother
+from .trajectory_smoother import TrajectorySmoother
```
Comment (FelixKHWong) [inline movo_common/movo_ros/src/movo_jtas/movo_jtas.py]: should this not be a file path?

**Review by FelixKHWong — COMMENTED** [review]
Seems good since just python fixes. However, why were so many files deleted here?? Did moveit get moved somewhere else or something?

**Review by le2tang — DISMISSED** [review]

**Review by FelixKHWong — DISMISSED** [review]
Just wondering but how do you know to delete these files?

File: movo_common/movo_ros/src/movo_jtas/movo_jtas.py (hunk 1 above)
Comment (younesr1) [reply movo_common/movo_ros/src/movo_jtas/movo_jtas.py]: both work in pyhthon3

**Comment by younesr1** [comment]
> Seems good since just python fixes. However, why were so many files deleted here?? Did moveit get moved somewhere else or something?

i deleted moveit_python and added it as a rosinstall dependency. see movo.rosinstall

**Comment by younesr1** [comment]
just rebased. can i get reapprovals

**Review by FelixKHWong — APPROVED** [review]

**Pull request merged**